operator-framework-junit5/src/main/java/io/javaoperatorsdk/operator/junit/OperatorExtension.java

```diff
@@ -275,12 +404,38 @@ public Builder withReconciler(Class<? extends Reconciler> value) {
       return this;
     }
 
+    public Builder withInfrastructure(List<HasMetadata> hm) {
```

Comment:
Would not be better with  `withInfrastructure(HasMetadata ... hm)` ?

Reply:
yup, done 👍